documentation/docs/developers/connectors/index.md

```diff
@@ -10,18 +10,21 @@ There are [existing connectors](https://github.com/CoinAlpha/hummingbot/tree/mas
 
 Building a new exchange connector requires conforming to the template code to the new exchange's APIs, identifying and handling any differences in functions/behaviors, and testing the new exchange connector on that exchange.
 
-!!! Tip: "Recommendation"
+
+!!! note "Recommendation"
     For new developers building a connector, we recommend to use Binance as a template to start and understand the functions and behaviors.
 
-- Centralized exchange: 
-    - [Binance](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/binance) 
-    - [Coinbase Pro](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/coin-base_pro) - [Huobi](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/huobi)
-    - [Bittrex](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/bittrex)
-    - Ethereum DEX (0x open order book): [Radar Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/radar_relay)
-    - Ethereum DEX (0x open order book w/coordinator support): [Bamboo Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/bamboo_relay)
+The following list some examples/templates that you can refer to when building the connector:
+
+- [Binance](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/binance) 
```

Comment:
Crypto.com connector should actually be the first example here as it is mainly Python implementation (others are mainly Cython) and will be easier for other developers to follow.

Reply:
![Screenshot 2020-10-05 at 9 18 38 AM](https://user-images.githubusercontent.com/52696508/95032085-d2e16480-06eb-11eb-9d1c-6905e7d6e1f0.png)
Noted and changed.